android/src/main/java/vn/luongvo/kmm/survey/android/util/ErrorMapping.kt

```diff
@@ -0,0 +1,14 @@
+package vn.luongvo.kmm.survey.android.util
+
+import android.content.Context
+import android.widget.Toast
+import vn.luongvo.kmm.survey.android.R
+
+fun Throwable.userReadableMessage(context: Context): String =
+    message ?: context.getString(R.string.generic_error)
+
+fun Throwable.showToast(context: Context) =
+    showToast(context, userReadableMessage(context))
+
+fun showToast(context: Context, message: String) =
```

Comment:
@luongvo It would be better if we put this function in ContextExt/FragmentExt

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/52/commits/ca8ac3cf7b436aa78b54ec7a90ba6f06bf5e93fd